protos/ray.proto

```diff
@@ -200,6 +201,19 @@ message GetObjRequest {
   uint64 objref = 1; // Object reference of the object being requested by the worker
 }
 
+message WorkerStatusRequest {
```

Comment:
These should be about tasks instead of workers, and for each task it should include the TaskId, the function name, the arguments, and the error message.